schema/types/base/height.go

```diff
@@ -7,19 +7,35 @@ package base
 
 import "github.com/persistenceOne/persistenceSDK/schema/types"
 
-type height struct {
-	Value int64 `json:"height"`
-}
+//type height struct {
+//	Value int64 `json:"height"`
+//}
+
+var _ types.Height = &Height{}
 
-var _ types.Height = (*height)(nil)
+func (height *Height) Get() int64 {
+	return height.Value
+}
 
-func (height height) Get() int64 { return height.Value }
-func (height height) IsGreaterThan(compareHeight types.Height) bool {
+func (height *Height) IsGreaterThan(compareHeight types.Height) bool {
```

Comment:
is it necessary to make it a pointer receiver?

Reply:
I used that because of the *types.Any, but after Ajaz found a way out, it can be done without pointer.